components/lvgl/Kconfig

```diff
@@ -25,22 +29,25 @@ menu "LittlevGL (LVGL) configuration"
 
     choice
 		prompt "Select a default font from the enabled built-in fonts"
-		default LVGL_DEFAULT_FONT_ROBOTO16
+		default LVGL_DEFAULT_FONT_MONTSERRAT16
 		help
 			Select a default font from the built-in fonts
 		
-		config LVGL_DEFAULT_FONT_ROBOTO12
-			select LVGL_FONT_ROBOTO12
-			bool "Roboto 12"
-		config LVGL_DEFAULT_FONT_ROBOTO16
-			select LVGL_FONT_ROBOTO16
-			bool "Roboto 16"
-		config LVGL_DEFAULT_FONT_ROBOTO22
-			select LVGL_FONT_ROBOTO22
-			bool "Roboto 22"
-		config LVGL_DEFAULT_FONT_ROBOTO28
-			select LVGL_FONT_ROBOTO28
-			bool "Roboto 28"
+		config LVGL_DEFAULT_FONT_MONTSERRAT12
+			select LVGL_FONT_MONTSERRAT12
+			bool "Montserrat 12"
+		config LVGL_DEFAULT_FONT_MONTSERRAT12
+			select LVGL_FONT_MONTSERRAT12SUBPIX
+			bool "Montserrat 12 sub-pixel"
+		config LVGL_DEFAULT_FONT_MONTSERRAT16
+			select LVGL_FONT_MONTSERRAT16
+			bool "Montserrat 16"
+		config LVGL_DEFAULT_FONT_MONTSERRAT22
+			select LVGL_FONT_MONTSERRAT22
+			bool "Montserrat 22"
+		config LVGL_DEFAULT_FONT_MONTSERRAT28
```

Comment:
We have fonts up to 48 px.  Is there any reason why the rest was skipped or this part is just not finished yet?

Reply:
It is not finished yet.